packages/events/src/util.ts

```diff
@@ -0,0 +1,66 @@
+/* Copyright (c) 2018 Environmental Systems Research Institute, Inc.
+ * Apache-2.0 */
+
+import { IRequestOptions } from "@esri/arcgis-rest-request";
+import { IItem } from "@esri/arcgis-rest-common-types";
+import { searchItems, ISearchResult } from "@esri/arcgis-rest-items";
+
+export interface IEventSearchResult extends ISearchResult {
```

Comment:
Do we need these interfaces? Can't we just use `ISearchResult` and `IItem` in this case?

Reply:
i'm guessing this is so TypeScript would quit complaining when she introspected for the `url` prop.

this is similar to what i did with [`IUserRequestOptions`](https://esri.github.io/arcgis-rest-js/api/auth/IUserRequestOptions/), but it's definitely worth a quick snoop to see if we can avoid it.